src/index.js

```diff
@@ -240,6 +240,12 @@ export default class Critters {
         .substring(pathPrefix.length)
         .replace(/^\//, '');
     }
+
+    // Ignore remote stylesheets
+    if (/^https:\/\//.test(normalizedPath) || href.startsWith('//')) {
```

Comment:
```suggestion
    if (/^https?:\/\//.test(normalizedPath) || href.startsWith('//')) {
```

Reply:
I'm guessing this was actually still doing `readFile('http://cdn.com/styles.css')` in the test, it's just emitting a soft warning (`Unable to locate stylesheet: ./http://cdn.com/styles.css`).

Reply:
Yeah you are right